content/about.html

```diff
@@ -18,7 +18,7 @@
 </style>
 <section>
 	<h1>Mission Statement</h1>
-	<p>GYC West is an organization of Seventh-day Adventist youth and young adults in the western United States. We believe that the Lord is forming His army of workers in these last days and our prayer is that GYC West would be a vessel through which Jesus Christ can reveal His will.</p>
```

Comment:
`Seventh-day Adventist` with the lower case 'day' is not a typo, that is the only correct form.